server/services/datacommons.py

```diff
@@ -62,7 +62,7 @@
     'get_place_obs': '/bulk/place-obs',
     'get_chart_data': '/node/chart-data',
     'get_stats': '/bulk/stats',
-    'get_related_places': '/node/related-places',
+    'get_related_places': '/node/related-locations',
```

Comment:
Is this also temporary, to support the rollout?

Reply:
Yeah, now there are both /node/related-places and /node/related-locations API and cache data. The old one will be removed after rollout.

Reply:
Cool. I'd prefer it if we kept /node/related-places at the end of the switch. :)

Reply:
okay, added a todo